visualize-admin/visualization-tool
# feat: Editor & chart tabs UI improvements

This PR:
- fixes ghost top rectangle in the homepage example charts,
- aligns editor and layouting page layouts,
- improves chart tabs scroll behavior,
- adds some Accent improvements.

Also see the [Figma file](https://www.figma.com/design/Hn9wvuEYLUmdJtg4KwZYUx/visualize.admin.ch---design?node-id=5240-95644&t=ZqPUv2ncXHZIaRlC-4).

## How to test
1. Go to [this link](https://visualization-tool-git-feat-dashboard-improvements-2-ixt1.vercel.app/en/create/new?cube=https://energy.ld.admin.ch/sfoe/bfe_ogd84_einmalverguetung_fuer_photovoltaikanlagen/10&dataSource=Prod).
2. Add a new chart.
3. Go to layouting step.
4. ✅ See that the general page layouts are aligned between both steps.
5. Go back to the editing mode.
6. Add more charts until they no longer fit on the screen (or resize browser window to achieve the same effect).
7. ✅ See that scroll buttons appeared and can be used to horizontally scroll the tabs.
